pkgs/development/libraries/qscintilla-qt4/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, lib, fetchurl, unzip
+, qt4 ? null, qmake4Hook ? null
+, withQt5 ? false, qtbase ? null, qtmacextras ? null, qmake ? null
```

Comment:
This can drop the Qt5 support
```suggestion
```

Reply:
I originally left this in to avoid breakage, but I think it's addressed by modifying all affected derivations still on Qt4.